gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FeatureEntity.java

```diff
@@ -134,10 +168,33 @@ public static Feature toFeature(FeatureEntity featureEntity, Project project) {
       return builder.build();
     }
 
+    if (featureEntity.getPolygonVertices() != null) {
+      PolygonFeature.Builder builder =
+          PolygonFeature.builder().setVertices(stringToList(featureEntity.getPolygonVertices()));
+      fillFeature(builder, featureEntity, project);
+      return builder.build();
+    }
+
     throw new LocalDataConsistencyException(
         "No geometry data found in feature " + featureEntity.getId());
   }
 
+  public static ImmutableList<Point> stringToList(String vertices) {
```

Comment:
Without context it's not clear that `stringToList()` operates on a list of vertices. How about `parsePolygonVertices()`?

Reply:
its converting the string to list that the reason this name was provided but parsePolygonVertices make more sense.

Reply:
renamed -> parsePolygonVertices